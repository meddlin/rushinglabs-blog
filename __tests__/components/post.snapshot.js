import { render } from '@testing-library/react';
import Post from '../../components/post';

it('renders SocialsHorizontal unchanged', () => {
    const { container } = render(<Post />);

    expect(container).toMatchSnapshot()
})