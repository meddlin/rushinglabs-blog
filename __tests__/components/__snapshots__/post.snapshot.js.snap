// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SocialsHorizontal unchanged 1`] = `
<div>
  <li
    class="listItem"
  >
    <div
      class="flex flex-col"
    >
      <div
        class="justify-center items-center"
      />
      <a
        href="/blog/undefined"
      />
    </div>
    <br />
    <small
      class="subpreview"
    >
      
      
    </small>
    <p />
    <a
      class="readMoreLink"
      href="/blog/undefined"
    >
      Read More â€”&gt;
    </a>
  </li>
</div>
`;
